commands/util/import.go

```diff
@@ -189,6 +190,11 @@ func ImportChain(ctx context.Context, r repo.Repo, fname string, snapshot bool)
 		return fmt.Errorf("importing chain failed: %w", err)
 	}
 
+	err = backfillMoreEpochsForTipsetKey(ctx, ts, cst, backfillTipsetkeyRange)
```

Comment:
Add comment to explain `cst.Import()` only backfills 1800 epochs.
Make the name shorter `backfillTipsetKey` .

Reply:
added more comment!